src/App.js

```diff
@@ -2,6 +2,8 @@ import "./App.css";
 import HomePage from "./Pages/HomePage/HomePage";
 import { Route, Routes } from "react-router-dom";
 import ProfilePage from "./Pages/ProfilePage/ProfilePage";
+import Login from "./Pages/AuthWidgets/Login/Login";
+import Signup from "./Pages/AuthWidgets/Signup/Signup";
```

Comment:
Suggestion- you can try barrel exports and export everything from Pages from single page.